readme.md

```diff
@@ -1,36 +1,38 @@
+- Manipulate the DOM using D3
 - Bind data to DOM elements
 - Scale data to fit the viewport
-- Use d3 helper methods like `min` and `max`
+- Use D3 helper methods like `min` and `max`
+
+## Framing (5 minutes / 0:05)
+
+### What Is D3?
 
-http://d3js.org/
-[Documentation](https://github.com/d3/d3/blob/master/API.md)
+D3 stands for Data-Driven Documents. The word "document" in D3 refers to the DOM. Javascript can manipulate the DOM, creating and deleting pieces of it. Building data visualizations, however, with tools that are suited for GUI programming is difficult and verbose. Enter D3.
 
-## What is it?
-D3 stands for Data-Driven Documents. The word "document" in D3 refers to the DOM. Javascript can manipulate the DOM, creating and deleting pieces of it. However! Building data visualizations with tools that are suited for GUI programming is difficult and verbose. Enter D3. More tools for building data visualizations.
+### What It's Not
 
-## What it's not
-D3 doesn't come with any prebuilt visualizations. D3 aims to be low-level and flexible. There are many excellent charting libraries out there, several of them built with D3. If your goal is to render a few standard charts as quickly as possible, you may want to consider using a library instead of writing D3 code by hand. Check out [highcartJS](http://www.highcharts.com/), [chartJS](http://www.chartjs.org/), or [dygraphs](http://dygraphs.com/).  
+D3 doesn't come with any prebuilt visualizations. D3 aims to be low-level and flexible. There are many excellent charting libraries out there, several of them built with D3. If your goal is to render a few standard charts as quickly as possible, you may want to consider using a library instead of writing D3 code by hand. Check out [highcartJS](http://www.highcharts.com/), [chartJS](http://www.chartjs.org/) or [dygraphs](http://dygraphs.com/).  
 
+### The Possibilities Are Endless
 
-## The Possibilities are Endless
+* http://www.jasondavies.com/animated-bezier/
+* http://bl.ocks.org/mbostock/1136236
+* http://animateddata.co.uk/lab/d3-tree/
+* http://d3tetris.herokuapp.com/
 
-- http://www.jasondavies.com/animated-bezier/
-- http://bl.ocks.org/mbostock/1136236
-- http://animateddata.co.uk/lab/d3-tree/
-- http://d3tetris.herokuapp.com/
+> D3 allows you to bind arbitrary data to  the DOM and then apply data-driven transformations to the document. For example, you can use D3 to generate an HTML table from an array of numbers. Or, use the same data to create an interactive SVG bar chart with smooth transitions.
 
->D3 allows you to bind arbitrary data to a Document Object Model (DOM), and then apply data-driven transformations to the document. For example, you can use D3 to generate an HTML table from an array of numbers. Or, use the same data to create an interactive SVG bar chart with smooth transitions and interaction.
+## You Do: [D3 Experiments in the Console](http://ga-wdi-exercises.github.io/d3-console/) (5 minutes / 0:10)
```

Comment:
Word, I'll add some questions that we'll talk about as a class after the exercise.